extension/src/prototypes/view/selection/MeshCodeObjectContent.tsx

```diff
@@ -68,7 +69,7 @@ export const MeshCodeObjectContent: FC<MeshCodeObjectContentProps> = ({ values }
     setSelection([]);
   }, [values, removeFeatures, setSelection]);
 
-  const features = useAtomValue(meshCodeLayers[0].featuresAtom);
+  const features = useAtomValue(meshCodeLayers[0]?.featuresAtom ?? fallbackAtom);
```

Comment:
I think you can use [useOptionalAtomValue](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/c92bf65a39a5ab64f44374f84eed79179cea0e5f/extension/src/shared/hooks/useOptionalAtom.ts#L16)

Reply:
Updated, thank you!